Dockerfile

```diff
@@ -8,6 +8,10 @@ RUN rm -rf ./*
 # Make sure the directory for individual app logs exists
 RUN mkdir -p /var/log/shiny-server
 
+# Install apt-get dependecies
+ADD apt-dependencies apt-dependencies
+RUN if [ -s apt-dependencies ]; then apt-get update && apt-get install -y $(cat apt-dependencies) ; fi
```

Comment:
Interesting, run `apt-get install` (without passing any list of packages) just work. So I guess we can avoid the check and just run `apt-get install` regardless:

    RUN apt-get update && apt-get install -y $(cat apt-dependencies)